public/js/main.js

```diff
@@ -67,4 +67,85 @@ $(document).ready(function () {
         e.preventDefault();
         window.location = "/api/csv/1";
     });
+
+    /**
+     * Functions for User List
+     */
+
+    $(".make_admin").click(function (e) {
+        e.preventDefault();
+        var activeUser = $(this).attr('data-user');
+        console.log('granting admin to ' + activeUser);
+        $.ajax({
+            type: 'POST',
+            url: '/api/account/roles',
+            data: {
+                email: activeUser,
+                roles: ['admin']
+            },
+            success: function (data, status) {
+                window.location.href = '/users';
+            }
+        });
+    });
+
+    $(".revoke_admin").click(function (e) {
+        e.preventDefault();
+        var activeUser = $(this).attr('data-user');
+        console.log('revoking admin from ' + activeUser);
+        $.ajax({
+            type: 'POST',
+            url: '/api/account/roles',
+            data: {
+                email: activeUser,
+                roles: []
+            },
+            success: function (data, status) {
+                window.location.href = '/users';
+            }
+        });
+    });
+
+    /**
+     * Functions for Project List
+     */
+
+    //Add a line to the table.
+    $("#add_project").click(function (e) {
```

Comment:
I think I just didn't notice that when I was doing a merge conflict resolution.  Should be better now.